BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -68,9 +68,9 @@ class CONTAINER_HDR (Structure):
     }
 
     _image_type = {
-      'NORMAL'     :  0x00,                 # Used for boot images in FV, regular ELF, PE32, etc. formats
-      'CLASSIC'    :  0xF3,                 # Used for booting Linux with bzImage, cmdline, initrd, etc.
-      'MULTIBOOT'  :  0xF4,                 # Multiboot compliant ELF images
+      'NORMAL'           :  0x00,                 # Used for boot images in FV, regular ELF, PE32, etc. formats
+      'CLASSIC_LINUX'    :  0xF3,                 # Used for booting Linux with bzImage, cmdline, initrd, etc.
```

Comment:
For the GenContainer parameter, let's just keep the same (CLASSIC) as it is now.

Reply:
Okay. Reverted the change for GenContainer.py